release/images.sh

```diff
@@ -36,6 +36,30 @@ sign_checksum() {
     fi
 }
 
+finalize_image() {
+    local name=${1}
```

Comment:
The interface for this function should provide the full path to the image, because that is how clr-installer will be doing it in the future.  I think this would be an easy change if we do something like:
```
local name_path="${1}"
local name=<some string substitution here to derive the name>
```

Then the rest of the code might be able to stay the same...

Reply:
updated to have name_path